src/components/input_fields/code_field.tsx

```diff
@@ -52,13 +50,20 @@ export default class CodeField extends React.Component<CodeFieldProps, CodeField
      * Note that we don't use React.CreateRef because that would require some extra code
      * for re-initializing the array whenever {maxLength} changes.
      */
-    private readonly _inputRefs: HTMLInputElement[] = [];
 
     /** The default props */
     static defaultProps = {
         type: 'number',
     };
 
+    static range(start: number, end: number): number[] {
```

Comment:
we could have this in a separate file so it can be reused

Reply:
moved